modules/administration/pages/monitoring.adoc

```diff
@@ -579,15 +605,72 @@ This section demonstrates how to generate client/server certificates for Prometh
 
 .Procedure: Creating server/client TLS certificate
 
-. On the {productname} Server, at the command prompt, run following command:
+. At the command prompt of the {productname} container host, as root, run the following commands:
+
++
+
+--
+
+.. To generate certificate files, run the following command.
+
++
+
+Ensure that the [systemitem]``set-cname`` parameter is the fully qualified domain name (FQDN) of your Salt client.
+You can use the the [systemitem]``set-cname`` parameter multiple times if you require multiple aliases:
+
 +
+
 ----
-rhn-ssl-tool --gen-server --dir="/root/ssl-build" --set-country="COUNTRY" \
---set-state="STATE" --set-city="CITY" --set-org="ORGANIZATION" \
---set-org-unit="ORGANIZATION UNIT" --set-email="name@example.com" \
---set-hostname="minion.example.com" --set-cname="minion.example.com" --no-rpm
+mgrctl exec -ti -- mgr-ssl-tool --gen-server --dir="/root/ssl-build" --set-country="COUNTRY" \
```

Comment:
Nice, so if you do that way, we don't really need to use `mgrctl term`

Reply:
I now only recommend `mgrctl term`  if the user must enter more commands in a row inside the container :^)